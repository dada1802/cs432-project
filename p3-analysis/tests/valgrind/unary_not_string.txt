==136852== Memcheck, a memory error detector
==136852== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==136852== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==136852== Command: ../decaf inputs/unary_not_string.decaf
==136852== 
Type mismatch: int expected but bool found on line 5
Type mismatch: bool expected but str found on line 9
==136852== 
==136852== HEAP SUMMARY:
==136852==     in use at exit: 0 bytes in 0 blocks
==136852==   total heap usage: 1,961 allocs, 1,961 frees, 234,456 bytes allocated
==136852== 
==136852== All heap blocks were freed -- no leaks are possible
==136852== 
==136852== For lists of detected and suppressed errors, rerun with: -s
==136852== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
