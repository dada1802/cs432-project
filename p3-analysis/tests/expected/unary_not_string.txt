Type mismatch: int expected but bool found on line 5
Type mismatch: bool expected but str found on line 9
